packages/hop-node/src/watchers/classes/L2Bridge.ts

```diff
@@ -228,6 +228,23 @@ export default class L2Bridge extends Bridge {
     }
   }
 
+  @rateLimitRetry
+  async getPendingTransferByIndex (chainId: number, index: number) {
+    return this.l2BridgeContract.pendingTransferIdsForChainId(
+      chainId,
+      index
+    )
+  }
+
+  async doPendingTransfersExist (chainId: number): Promise<boolean> {
+    try {
+      await this.getPendingTransferByIndex(chainId, 0)
+      return true
+    } catch (err) {
+      return false
```

Comment:
In the case of an unrelated RPC error, this will always return false when there could be a pending transfer. 
The error message should be filtered